src/policy/policy.h

```diff
@@ -0,0 +1,115 @@
+#ifndef SRC_POLICY_POLICY_H_
+#define SRC_POLICY_POLICY_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "v8.h"
+
+#include <bitset>
+#include <string>
+#include <vector>
+
+namespace node {
+
+class Environment;
+
+namespace policy {
+
+#define PERMISSIONS(V)                                                         \
+  V(Special, "special", PermissionsRoot)                                       \
+  V(SpecialInspector, "special.inspector", Special)                            \
+  V(SpecialAddons, "special.addons", Special)                                  \
+  V(SpecialChildProcess, "special.child_process", Special)                     \
+  V(Workers, "workers", PermissionsRoot)                                       \
```

Comment:
We should make sure that Workers start out with the same policy as the spawning context, not just the ones passed to Environment in the initial setup through `execArgv` – I think #22112 did that.

Reply:
Yep, going to tackle that when I work on the worker specific policy stuff. Specifically, when a `Worker` is created, it will inherit the current policy or the provided policy.